doc/sphinx/develop/distribution-packages/pypi-sdist-wheel.md

```diff
@@ -143,15 +150,13 @@ When a version of Cantera is ready to be published to PyPI, the workflow in the
 [pypi-packages](https://github.com/Cantera/pypi-packages) repository must be executed.
 The workflow essentially conducts the steps in the [](#tldr) section above.
 
-The one big addition to the GitHub Action workflow is that there are a couple of shell
-scripts in the `pypi-packages` repository to build libhdf5 for macOS and Windows. On
-macOS, we also link to `libaec` so there's a small patch for the CMakeLists.txt file in
-`libaec`.
+The one big addition to the GitHub Actions workflow is that there are a couple of shell
+scripts in the `pypi-packages` repository to build libhdf5, HighFive, and SUNDIALS for
+macOS and Windows.
 
-For Linux builds, we have a [custom-built manylinux
-image](https://github.com/Cantera/hdf5-boost-manylinux) that already has the
-dependencies installed. This saves time compiling libhdf5 on Linux/ARM builds because
-those are emulated and already very slow.
+For Linux builds, we have a [custom-built manylinux image][manylinux] that already has the
+dependencies installed. This saves time compiling our dependencies on Linux/ARM builds
```

Comment:
I find the end-of-document link targets a bit confusing -- we don't use them elsewhere.

Reply:
I used them to have shorter line lengths since we're breaking at ~88 characters. I can pull them out.